File: 程式語言 V2答案檔/Python/505.py
import math
from decimal import Decimal, ROUND_HALF_UP

input_float = []
for i in range(6):
    input_float.append(float(input()))
ans = abs(input_float[0]) * math.floor(input_float[1]) + (input_float[2] ** input_float[3]) * math.sqrt(input_float[4]) + math.log(input_float[5], 10)
# print(ans)
ans_round = Decimal(str(ans)).quantize(Decimal('.00'), rounding = ROUND_HALF_UP)
print(ans_round)

File: 程式語言 V2答案檔/Go/505.go
package main
import (
	"fmt"
	"math"
)

func main(){
	var a, b, c, d, e, f float64
	fmt.Scanln(&a)
	fmt.Scanln(&b)
	fmt.Scanln(&c)
	fmt.Scanln(&d)
	fmt.Scanln(&e)
	fmt.Scanln(&f)
	formula := math.Abs(a) * math.Floor(b) + math.Pow(c, d) * math.Sqrt(e) + math.Log10(f)
	fmt.Printf("%.2f\n", formula)
}
